oraclepaas/resource_mysql_service_instance.go

```diff
@@ -0,0 +1,662 @@
+package oraclepaas
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	opcClient "github.com/hashicorp/go-oracle-terraform/client"
+	"github.com/hashicorp/go-oracle-terraform/mysql"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceOraclePAASMySQLServiceInstance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOraclePAASMySQLServiceInstanceCreate,
+		Read:   resourceOraclePAASMySQLServiceInstanceRead,
+		Update: resourceOraclePAASMySQLServiceInstanceUpdate,
```

Comment:
It looks like everything is ForceNew so let's remove this and the respective method down below

Reply:
Done